http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAO.scala

```diff
@@ -210,7 +210,7 @@ object HttpDockerDAO {
     val res = for {
       splitted <- Either.catchNonFatal(s.split("="))
       first <- Either.catchNonFatal(splitted(0))
-      second <- Either.catchNonFatal(splitted(1))
+      second <- Either.catchNonFatal(if (splitted.length > 1) splitted(1) else "")
```

Comment:
Not related to this ticket, but a user found this so thought I'd fix. It's caused by EVs in the Docker image with a blank value (e.g. `FOO=`).